src/script/lua_api/l_client.cpp

```diff
@@ -414,6 +414,26 @@ int ModApiClient::l_get_csm_restrictions(lua_State *L)
 	return 1;
 }
 
+// setting_get(setting_name)
+// The whole LuaSettings API (which allows for setting modification) isn't
+// exposed to CSMs
+int ModApiClient::l_setting_get(lua_State *L)
+{
+	std::string key = std::string(luaL_checkstring(L, 1));
+	// Disallow reading some settings
```

Comment:
Ant settings started with `secure.`

Reply:
Those settings should be blocked with the check for `.` in the setting name.

Reply:
What about mesecon. setting? Or mobs. ?

Reply:
Those will be blocked as well. I don't see any reason for SSCSMs to be able to read those settings from the client's configuration.